plugins/modules/source_control/gitlab/gitlab_group_members.py

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/python
```

Comment:
A general remark: you need to create a symlink from `plugins/modules/` to your module.

Reply:
I tested the modules locally by writing playbooks and running them against a local gitlab server in a container. 

however, I wrote integration test. please let me know if it needs any modifications.